client/state/data-layer/wpcom/concierge/schedules/appointments/detail/index.js

```diff
@@ -0,0 +1,54 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { translate } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import { http } from 'state/data-layer/wpcom-http/actions';
+import { dispatchRequest } from 'state/data-layer/wpcom-http/utils';
+import { errorNotice } from 'state/notices/actions';
+import { updateConciergeAppointmentDetails } from 'state/concierge/actions';
+import { CONCIERGE_APPOINTMENT_DETAILS_REQUEST } from 'state/action-types';
+import { noRetry } from 'state/data-layer/wpcom-http/pipeline/retry-on-failure/policies';
+import fromApi from './from-api';
+
+export const fetchAppointmentDetails = ( { dispatch }, action ) => {
+	const { appointmentId, scheduleId } = action;
+
+	dispatch(
+		http(
+			{
+				method: 'GET',
+				path: `/concierge/schedules/${ scheduleId }/appointments/${ appointmentId }/detail`,
+				apiNamespace: 'wpcom/v2',
+				retryPolicy: noRetry(),
+			},
+			action
+		)
+	);
+};
+
+export const storeFetchedAppointmentDetails = ( { dispatch }, action, appointmentDetails ) => {
+	dispatch( updateConciergeAppointmentDetails( appointmentDetails ) );
+};
+
+export const showAppointmentDetailsFetchError = ( { dispatch } ) => {
+	dispatch(
+		errorNotice( translate( 'We could not find your appointment. Please try again later.' ) )
```

Comment:
Hi! I've found a possible matching string that has already been translated **18** times:
`translate( 'We could not book your appointment. Please try again later.' )` *ES Score: 9*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).